openedx/features/edly/models.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.sites.models import Site
+from django.core.validators import RegexValidator
+from django.db import models
+from organizations.models import Organization
+
+ALPHANUMERIC_VALIDATOR = RegexValidator(r'^[0-9a-zA-Z]*$', 'Only alphanumeric characters are allowed.')
+
+
+class EdlyOrganization(models.Model):
+    name = models.CharField(max_length=255)
+    slug = models.CharField(max_length=100, unique=True, validators=[ALPHANUMERIC_VALIDATOR])
+
+    def __str__(self):
+        return u"{name} ({slug})".format(name=self.name, slug=self.slug)
+
+
+class EdlySubOrganization(models.Model):
+    edly_organization = models.ForeignKey(EdlyOrganization, null=False, blank=False)
+    edx_organization = models.OneToOneField(Organization)
+    lms_site = models.OneToOneField(Site, related_name='lms_site')
```

Comment:
no need for `related_name` here

Reply:
We need to add this related name here.
<img width="1302" alt="Screenshot 2020-05-04 at 12 43 20 PM" src="https://user-images.githubusercontent.com/17013371/80945281-e4ed6d00-8e04-11ea-8b0a-1ad4a63fcd61.png">

Reply:
@zubair-arbi  we need `related_name` here.
<img width="1285" alt="Screenshot 2020-05-04 at 12 50 37 PM" src="https://user-images.githubusercontent.com/17013371/80945725-e10e1a80-8e05-11ea-8c79-bace5e420b1d.png">

Reply:
Ok. Lets keep the `related_name`.